projects/core/src/product/store/reducers/product-search.reducer.ts

```diff
@@ -36,7 +37,7 @@ export function reducer(
     }
 
     case fromProductsSearch.GET_PRODUCT_SUGGESTIONS_SUCCESS: {
-      const suggestions = action.payload;
+      const suggestions: Suggestion[] = action.payload;
 
       return {
         ...state,
```

Comment:
Line 55: 
```
export const getSearchResults = (
  state: ProductsSearchState
): ProductSearchPage => state.results;
```
line 58:
```
export const getSearchResults = (
  state: ProductsSearchState
): ProductSearchPage => state.results;
```

Reply:
@LTiger14 the Line 55 changes causes errors. `type Review[] has no properties in common with ProductSearchPage`

Reply:
😉😉😉😉😉😉😉